ray-kubernetes/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:latest
+
+# If you wish to default to bash shell instead of sh(Bourne) shell:
+#     RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+# disable interactive functions, suppress tzdata prompting for geographic location
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt -qq update \
+    && apt-get -qq install --no-install-recommends apt-utils \
+    && apt-get -qq install git wget \
+    && apt-get -qq install cmake pkg-config build-essential autoconf curl libtool unzip flex bison psmisc nano openssh-server \
+    && apt-get -qq install python-opencv
+
+# Install miniconda3, create env named: ray-kubernetes
+RUN curl -LO http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+RUN bash Miniconda3-latest-Linux-x86_64.sh -p /miniconda3 -b
+RUN rm Miniconda3-latest-Linux-x86_64.sh
+ENV PATH=/miniconda3/bin:${PATH}
+RUN conda update -y conda
+RUN conda create -y -n ray-kubernetes python=3.6.5
```

Comment:
Are you creating a conda environment just to change the python version? Is there any other advantage to using conda here? If not, I think it'd be simpler to not use conda environments

Reply:
In that case, should we use python2 instead of python3? 
I just found it messy specifying the aliases for pip and python without conda, and I guess my thought was conda would make it easier for whatever future updates.
But let me know which python version you think should be used, and i'll update and test the new image accordingly

Reply:
sorry, I misuderstood, thought you meant not to use conda at all. you were just referring to the conda enviroment `ray-kubernetes`
This is actually needed since newest conda ships with `python 3.7` and `pip` has issues in that version. So unless we use `miniconda` instead of `miniconda3` it is necessary to change the python version

Reply:
I'd suggest either downloading miniconda in version 3.6, e.g., 

```
wget https://repo.continuum.io/miniconda/Miniconda3-4.5.4-Linux-x86_64.sh -O miniconda.sh -nv
bash miniconda.sh -b -p $HOME/miniconda
export PATH="$HOME/miniconda/bin:$PATH"
```

Or using 3.7 and installing Ray via

```
pip install https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.5.3-cp37-cp37m-manylinux1_x86_64.whl
```